### PR TITLE
Logger implementation

### PR DESCRIPTION
**Pull request by li46554**
Added logger to classes

### CODE REVIEW

**Review by Henrytoo — CHANGES_REQUESTED** [review]
The levels for the logging messages are not the correct levels. Check the documentation for Logging so that you know which ones should be used. In addition, these logs need to be more specific and self-descriptive of what they are for, You will need to check old code to know how to format these new logs.

File: RobotCode/src/main/java/frc/robot/commands/ClimberCommand.java
Comment (PPenguin1) [inline RobotCode/src/main/java/frc/robot/commands/ClimberCommand.java:1]: This is not needed

**Review by PPenguin1 — APPROVED** [review]

**Review by Henrytoo — CHANGES_REQUESTED** [review]
Again these logs need to be more specific so that we can know exactly where the problem is without any confusion.

**Review by seankid — APPROVED** [review]
mucho bueno alberto. es muy brazo.

**Pull request closed without merging**